red4sec audit summary.md

```diff
@@ -0,0 +1,55 @@
+# Red4Sec audit summary
+
+- Audit length: 7 weeks. 
+- Auditor homepage: https://red4sec.com
+
+Red4Sec’s report was exhaustive, and covered the entirety of our project space, with the exception of our post-quantum cryptography, which was coverd by x41 D-Sec. Below is a rundown of issues raised by Red4Sec by section of our network, as well as impact level.
```

Comment:
“Coverd” typo could become “reviewed separately”

Reply:
Good catch and suggestion. Change made.